config/config.js

```diff
@@ -17,7 +17,7 @@
 
 // Hierarchical node.js configuration with command-line arguments, environment
 // variables, and files.
-const nconf = module.exports = require('nconf');
```

Comment:
Seems to be more conventional (at least in our project) to do this at the bottom of the file.